src/components/map/MapProvider.js

```diff
@@ -1,6 +1,8 @@
 import { Component } from 'react';
 import PropTypes from 'prop-types';
-import d2map from '@dhis2/gis-api';
+// import D2map from '@dhis2/gis-api';
+import D2map from '@dhis2/gis-api/src';
```

Comment:
This should use the `import D2map from '@dhis2/gis-api'` form, which works if you build `gis-api` before linking it

Reply:
See the steps I took to link (which may be incorrect?  In [gis-api#45](https://github.com/dhis2/gis-api/pull/45)